src/main/kotlin/demo/controllers/OrganizationController.kt

```diff
@@ -0,0 +1,66 @@
+package demo.controllers
+
+import demo.models.people.Organization
+import demo.services.OrganizationService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.util.*
+import java.util.NoSuchElementException
+
+@RestController
+@RequestMapping("/api/organizations")
+class OrganizationController(@Autowired private val organizationService: OrganizationService) {
+    @GetMapping
+    fun getOrganizations(): ResponseEntity<MutableIterable<Organization>> {
+        return ResponseEntity(organizationService.getOrganizations(), HttpStatus.OK)
+    }
+
+    @GetMapping("/{organizationId}")
+    fun getOrganization(@PathVariable organizationId: OrganizationID): ResponseEntity<Organization> {
+        val organization: Organization = organizationService.getOrganization(organizationId.toId())
+
+        return  ResponseEntity(organization, HttpStatus.OK)
+    }
+
+    @PostMapping("/addorganization")
+    fun addOrganization(@RequestBody organization: OrganizationRequest): ResponseEntity<Organization> {
+        val saved: Organization = organizationService.addOrganization(organization.toOffer())
+
+        val httpHeaders = HttpHeaders()
+        httpHeaders.add(HttpHeaders.ACCEPT, "/api/organizations/" + saved.id)
+        return ResponseEntity(httpHeaders, HttpStatus.CREATED)
```

Comment:
Wydaje mi się, że tutaj chodziło o header Location, a nie Accept. Przy okazji możnaby te 3 linijki skompresować.
```suggestion
        return ResponseEntity.created(URI("/api/organizations/" + saved.id)).build()
```